### ISSUE TITLE
Sync Erro

### ISSUE THREAD

**Issue by Legion75** [Woprok/AOE4-AdvancedGameSettings#9]
Hi, i got a problem, every time i create a custom game with the mod i get a sync-error, it has a fix or i cant use de mod on multiplayer? thanks

**Comment by Woprok** [comment]
@Legion75 can you provide more info about selected settings?

It is supposed to work in mp

**Comment by Woprok** [comment]
I uploaded new version, that fixes another issue, which might be related to this, so try it again and give detailed description if problem appears again.

**Comment by Legion75** [comment]
i tried alone with bots and new version and worked

**Comment by Woprok** [comment]
@Legion75 yeah there was issues with bots in multiplayer game, I do not have a way to test real multiplayer game as adding bots to mp lobby does not count unless another human player is involved.

**Comment by Woprok** [comment]
@Legion75 Its fixed, bots should no longer crash multiplayer

**Comment by Woprok** [comment]
if any problems appear let me know
